src/qz/installer/Installer.java

```diff
@@ -170,6 +170,37 @@ public Installer removeLibs() {
         return this;
     }
 
+    public Installer cleanupLegacyLogs(int rolloverCount) {
+        // Convert old < 2.2.3 log file format
+        Path logLocation = USER_DIR;
+        int oldIndex = 0;
+        int newIndex = 0;
+        while(true) {
+            // Old: debug.log.1
+            File oldFile = logLocation.resolve("debug.log." + oldIndex++).toFile();
+            if(oldFile.exists()) {
+                // Rename to new format
+                inner:
+                while(true) {
+                    // New: debug.1.log
+                    File newFile = logLocation.resolve("debug." + ++newIndex + ".log").toFile();
+                    if (!newFile.exists()) {
+                        oldFile.renameTo(newFile);
+                        log.info("Migrated log file {} to new location {}", oldFile, newFile);
+                        break inner;
+                    }
+                }
+                continue;
+            }
+            // Loop at least as high as our rollover strategy in case there are human made gaps
+            if(oldIndex <= rolloverCount) continue;
+            break;
```

Comment:
this is a bit confusing to read, instead can we use a do..while that loops if oldFile exists or index is less than rollover?
```
do { .. } while(oldFile.exists() || oldIndex <= rolloverCount);
```

also, pretty sure break/continue only works on the innermost loop, so the "inner" label in unnecessary